ci-operator/step-registry/telcov10n/sno-hub-ztp/deploy-hub/telcov10n-sno-hub-ztp-deploy-hub-chain.yaml

```diff
@@ -1,9 +1,10 @@
 chain:
-  as: telcov10n-hub-spoke-gitops-ztp-deploy-hub
+  as: telcov10n-sno-hub-ztp-deploy-hub
   steps:
       - chain: baremetal-lab-agent-conf
       - ref: baremetal-lab-agent-conf-dynamic
       - chain: baremetal-lab-agent-install
+      - ref: baremetal-set-preserved
```

Comment:
@aleskandro the new step will be added here, right after the cluster installation is done.